miranda/structure/data/ouranos_schema.yml

```
@@ -0,0 +1,61 @@
+---
+datasets:
+  - option: type
+    value: station-obs
+    structure:
+      - type
+      - institution
+      - source
+      - version
+      - frequency
+      - option: member
+        is_true: member
+      - variable
+  - option: type
+    value: forecast
+    structure:
+      - type
+      - institution
+      - source
+      - domain
+      - frequency
+      - variable
+  - option: type
+    value: gridded-obs
+    structure:
+      - type
+      - institution
+      - source
+      - domain
+      - frequency
+      - variable
+  - option: type
+    value: reconstruction
+    structure:
+      - type
+      - institution
+      - source
+      - domain
+      - frequency
+      - variable
+  - option: type
+    value: simulation
+    structure:
+      - type
+      - processing_level
+      - mip_era
+      - activity
+      - option: processing_level
+        value: raw
+        is_true: processing_level
+        else: bias_adjust_project
```

```suggestion
     - option: processing_level
        value: biasadjusted
        is_true: bias_adjust_project
      - mip_era
      - activity
```
As suggested in slack.

---

@juliettelavoie @tlogan2000 @RondeauG How do we feel about this?

---

ok with me!

---

Good for me too

---

Alrighty